### PR TITLE
Added zap stanza for QtPass v1.2.3

### PR DESCRIPTION
**Pull request by thirdbody**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [ ] Named the cask according to the [token reference].
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked there are no [open pull requests] for the same cask.
- [ ] Checked the cask was not [already refused].
- [ ] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/qtpass.rb [hunk 1]
```diff
@@ -9,4 +9,6 @@
   homepage 'https://qtpass.org/'
 
   app 'QtPass.app'
+
+  zap trash: '~/Library/Saved Application State/$(PRODUCT_BUNDLE_IDENTIFIER).savedState'
```
Comment (vitorgalvao) [inline Casks/qtpass.rb]: You forgot to replace `$(PRODUCT_BUNDLE_IDENTIFIER)`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/qtpass.rb (hunk 1 above)
Comment (Myl0g) [reply Casks/qtpass.rb]: I didn't actually; QtPass's directory is literally `$(PRODUCT_BUNDLE_IDENTIFIER)`. That directory is created every time QtPass is quit.

Comment (vitorgalvao) [reply Casks/qtpass.rb]: That is a clear error, and one waiting for an accident to happen. Let’s [wait for a fix](https://github.com/IJHack/QtPass/issues/448).

Comment (Myl0g) [reply Casks/qtpass.rb]: Haha, good call. Thanks for opening up the issue.

Comment (vitorgalvao) [reply Casks/qtpass.rb]: Issue has been open for 20 days, and last commit was in December. I’m not sure anyone is checking the project.

Thank you for your contribution. I’ll close as that ID is potentially dangerous. We’ll revisit when/if they fix it on their end.

**Pull request closed without merging**